static/app.template.html

```diff
@@ -29,83 +26,95 @@
 		<script src="//cdnjs.cloudflare.com/ajax/libs/flot.tooltip/0.8.5/jquery.flot.tooltip.js"></script>
 		<script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha/js/bootstrap.min.js"></script>
 		<script src="//cdnjs.cloudflare.com/ajax/libs/tether/1.3.2/js/tether.min.js"></script>
-		<script type="text/javascript">
-var amountSummaryData = [[{{range .AmountSummaryChart}}{{.}},{{end}}]];
-var interestRateData = [[{{range .InterestRateChart}}{{.}},{{end}}]];
-function formatNumber(n, preSymbol, postSymbol) {
-  nStr = preSymbol + ' ' + n.toFixed(2);
-  x = nStr.split('.');
-  x1 = x[0];
-  x2 = x.length > 1 ? '.' + x[1] : '';
-  var rgx = /(\d+)(\d{3})/;
-  while (rgx.test(x1)) {
-    x1 = x1.replace(rgx, '$1' + ',' + '$2');
-  }
-  return x1 + x2 + ' ' + postSymbol;
-}
+		<script src="//cdnjs.cloudflare.com/ajax/libs/numeral.js/1.5.3/numeral.min.js"></script>
+
+<script type="text/javascript">
+// Loading language.
+// Extracted from github.com/adamwdraper/Numeral-js/blob/master/languages/pt-br.js
+numeral.language('pt-br', {
+  delimiters: {
+    thousands: '.',
+    decimal: ','
+  },
+  abbreviations: {
+    thousand: 'mil',
+    million: 'milhões',
+    billion: 'b',
+    trillion: 't'
+  },
+  ordinal: function (number) {
+    return 'º';
+  },
+  currency: {
+    symbol: 'R$'
+  },
+})
+
+// Setting language and default currency formatter.
+numeral.language('pt-br')
+numeral.defaultFormat('$ 0[.]0[,]00');
 
 function formatCurrency(n) {
-  return formatNumber(n, '{{.Currency}}', '')
+  return numeral(n.toFixed(2)).format()
 }
 
 function formatPercent(n) {
-  return formatNumber(n, '', '%')
+  return numeral(n).format('0,00%')
```

Comment:
Now, the interval of the yaxis of the Interest Rate is between 0% and 100%. Before that, was between -1 and 1.